machine_learning/xptracking_catboost/code.py

```diff
@@ -0,0 +1,82 @@
+import dataiku
+import mlflow
+import mlflow.catboost
+import pandas as pd
+from catboost import CatBoostClassifier, Pool, cv
+from sklearn.model_selection import train_test_split
+from sklearn.preprocessing import LabelBinarizer
+from sklearn import metrics
+from dataikuapi.dss.ml import DSSPredictionMLTaskSettings
+
+# !! - Replace these values by your own - !!
+USER_PROJECT_KEY = "YOUR-PROJECT-KEY"
+USER_XPTRACKING_FOLDER_ID = "YOUR-XP-TRACKING-FOLDER-ID"
+USER_EXPERIMENT_NAME = "YOUR-EXPERIMENT-NAME"
+USER_TRAINING_DATASET = "YOUR-TRAINING-DATASET-NAME"
+USER_MLFLOW_CODE_ENV_NAME = "YOUR-MLFLOW-CODE-ENV-NAME"
+
+client = dataiku.api_client()
+project = client.get_project(USER_PROJECT_KEY)
+
+# (1)
+ds = dataiku.Dataset(USER_TRAINING_DATASET)
+df = ds.get_dataframe()
+
+cat_features= ["job", "marital", "education", "default", "housing","loan", "month", "contact", "poutcome"]
+num_features= ["age", "balance", "day", "duration", "campaign"]
+target_col ="y"
+X = df.drop(target_col, axis=1)
+y = LabelBinarizer().fit_transform(df[target_col[0]])
```

Comment:
no need to transform target column for catboost, so can be changed to
`y = df[target]`